cmd/compute-domain-controller/daemonset.go

```diff
@@ -334,11 +349,6 @@ func (m *DaemonSetManager) onAddOrUpdate(ctx context.Context, obj any) error {
 		return fmt.Errorf("failed to cast to DaemonSet")
 	}
 
-	// Only process events from the driver namespace
-	if d.Namespace != m.config.driverNamespace {
-		return nil
-	}
-
```

Comment:
I am sure this was removed for good reason, but I am not entirely sure I follow why. Was this check removed since the informer for each `DaemonSetManager` is now namespaced, and therefore, the object we are processing at this point will always be in `m.config.driverNamespace`?